requirements.txt

```diff
@@ -0,0 +1,3 @@
+rasa==2.8.0
```

Comment:
I think I prefer pinned requirements vs the unpinned to pinned situation

Reply:
so, I don't really get what you mean here? (I didn't use our whole unpinned flow, just provided a basic requirements.txt)

Reply:
👍